quinn/src/runtime/mod.rs

```diff
@@ -12,6 +12,9 @@ use udp::{RecvMeta, Transmit};
 
 use crate::Instant;
 
+#[cfg(not(any(feature = "runtime-tokio", feature = "runtime-smol", feature = "custom-runtime")))]
+compile_error!("no async runtime found. Please enable a runtimse such as runtime-tokio, runtime-smol or custom-runtime");
```

Comment:
typo: "runtimse". Let's also word this as "a runtime feature" to provide a clearer breadcrumb to users who might not know what we're talking about.

Also please be consistent about capitalization and punctuation: if the text contains multiple sentences, then the first letter should be capitalized and there should be a trailing period.